src/data/nat/basic.lean

```diff
@@ -39,6 +39,12 @@ begin
     rw [max_eq_left h2, max_eq_left (succ_le_succ h2)] }
 end
 
+lemma not_succ_lt_zero : ∀ (n : ℕ), succ n < 0 → false
```

Comment:
This is redundant. See `nat.not_lt_zero`

Reply:
Removed. I must be tired when I wrote this... Thanks!